src/main/java/gillycarpetaddons/mixins/FlowerBlock_replaceFlowersMixin.java

```diff
@@ -0,0 +1,38 @@
+package gillycarpetaddons.mixins;
+
+import net.minecraft.block.Block;
+import net.minecraft.block.BlockState;
+import net.minecraft.block.FlowerBlock;
+import net.minecraft.block.Material;
+import net.minecraft.item.ItemPlacementContext;
+import net.minecraft.item.ItemStack;
+import net.minecraft.util.math.BlockPos;
+import net.minecraft.world.World;
+import org.spongepowered.asm.mixin.Mixin;
+
+import static gillycarpetaddons.GillyCarpetAddonsSettings.replaceableFlowers;
+
+@Mixin(FlowerBlock.class)
+public abstract class FlowerBlock_replaceFlowersMixin extends Block {
+
+  //makes ide happy
```

Comment:
Doesn't make just the IDE happy. Block class requires a constructor, so the construction is needed.

Also, please use java doc strings instead of single line comments. I.e. /**

Here we don't need the doc string since it is a single constructor.

Reply:
Thats the reason it need but since class is abstract it will never be used, it makes it compile and makes ide happy but serves no actual reason as it will never be runned.